index.js

```diff
@@ -56,11 +56,7 @@ export default class RNUrlPreview extends React.PureComponent {
   };
 
   componentWillReceiveProps(nextProps) {
```

Comment:
Please change this to componentDidUpdate. As of React 16, componentWillReceiveProps throws a warning that it is being deprecated. It will be removed in React 17. The suggested fix is handle this is to either change to componentDidUpdate or use hooks.